canonical-web-and-design/jaas.ai
# Revert assets change

## Done
Revert assets change back to static
Add redirect from assets to static

## QA
- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8029
- Click around and see it works
- Go to /assets/blah and check it redirects to /static/blah
